app.js

```diff
@@ -10,9 +10,7 @@ const dbConfig = require('./database/db');
 var cookieparser =require('cookie-parser');
 const csurf = require('csurf');
 var multer = require('multer');
-var upload = multer({
-  limits: { fileSize: 1024*1024 }
-});
+var upload = multer();
```

Comment:
#### Automated PR Comment From coverity
<!-- c60191972934886a7415103a01b14cfe -->
<!-- Comment managed by Git Commenter, do not modify! -->
### Coverity Issue - File upload with unrestricted filesize
_High CWE-400_
The `multer` module does not restrict the size of the uploaded file. Thus, a denial-of-service attack is possible, because an attacker may submit a series of requests with very large files and use up all the space on the server.
### How to fix
Limit the size of an uploaded file by setting the `fileSize` property to a reasonable number no greater than 500MB in the `limits` object of the `multer` options.